cmd/fleet/metrics.go

```diff
@@ -36,9 +36,12 @@ var (
 
 func (f *FleetServer) initMetrics(ctx context.Context, cfg *config.Config) (*api.Server, error) {
 	registry := monitoring.GetNamespace("info").GetRegistry()
-	monitoring.NewString(registry, "version").Set(f.ver)
-	monitoring.NewString(registry, "name").Set("fleet-server")
-	metrics.SetupMetrics("fleet-server")
+	if registry.Get("version") == nil {
+		monitoring.NewString(registry, "version").Set(f.ver)
+	}
+	if registry.Get("name") == nil {
+		monitoring.NewString(registry, "name").Set("fleet-server")
+	}
```

Comment:
This was required to be changed, because it cannot be called more than once per process. But the version is not know until it reaches this far.

If it is called more than once it will panic.